### DISCUSSION TITLE
[Question/Info Request] Gameplay affecting input delay on Windows 11.

### DISCUSSION THREAD

**Discussion by LifeDreamer24** [redphx/better-xcloud#60] [General]
Despite having a good connection in my opinion (3gb/s for my household), i feel that there is an annoying fraction of a second delay that confuses me a lot with my controls and my internet speed. Comparing to GeForce NOW, there is a huge difference to mention. No delay seems to be seen there on my side. Is it because this streaming service is hosting Computers and Xbox Cloud Gaming is doing the same but for consoles which affects the quality of streaming or it's purely my problem? Am i alone being affected by this or there are similar cases with other people using a good connection speed?

I am aware that this doesn't relate to this project, although i hope some kind being could help me solve or improve my confusing situation.

Thanks in advance,
> Simon

**Comment by redphx** [comment]
What do your stream stats look like? And is your computer fast?

**Comment by LifeDreamer24** [comment]
I assume so.. Let me check and I'll send you the most information possible. 😄

**Comment by redphx** [comment]
Also check the stats in `chrome://webrtc-internals`

**Comment by LifeDreamer24** [comment]
![image](https://github.com/redphx/better-xcloud/assets/62817314/a01756f8-7cd2-4d48-b001-a4b30221cbcc)
![image](https://github.com/redphx/better-xcloud/assets/62817314/6c8ac776-431f-4e9f-92a8-59f0693f6b67)
![image](https://github.com/redphx/better-xcloud/assets/62817314/e6bcb677-ba79-4603-a197-9e4d99cfc6e1)
![image](https://github.com/redphx/better-xcloud/assets/62817314/8d0c2d9f-3818-4d26-869d-342c4a94a8d0)
![image](https://github.com/redphx/better-xcloud/assets/62817314/ecab5103-70c1-475e-ac5d-1ee0952d186a)
![image](https://github.com/redphx/better-xcloud/assets/62817314/54a6123a-d37b-463c-a947-1519f04706de)
![image](https://github.com/redphx/better-xcloud/assets/62817314/c5f4cc95-967a-4b94-b160-2f2b53a0f929)
